### DISCUSSION TITLE
MÓDULO 0 / PREPARACIÓN

### DISCUSSION THREAD

**Discussion by CarlosPena97** [LaunchX-InnovaccionVirtual/CursoIntroPython#305] [Q&A]
Trato de crear el botón "Iniciar Launch" mediante el código que aparece en elmódulo: 

import ipywidgets as widgets

ignition = widgets.ToggleButton(
    value=False,
    description='Iniciar Launch',
    button_style='success',
    tooltip='Engage your Launch',
    icon='rocket'
)

output = widgets.Output()

display(ignition, output)

def on_value_change(change):
    with output:
        if change['new'] == True:
            print("Nave Iniciada!")
        else:   
            print("Nave Detenida")

ignition.observe(on_value_change, names='value')

Pero me marca el siguiente error:

Input In [5]
    pip install ipywidgets
        ^
SyntaxError: invalid syntax

**Comment by CarlosPena97** [comment]
Ya pude, gracias. Estoy empezando.

**Comment by doguedogue** [comment]
Genial, lo solucionaste
